<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailsWindowFlightRental</class>
 <widget class="QDialog" name="DetailsWindowFlightRental">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>618</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="IDLabel">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ID">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="PriceLabel">
       <property name="text">
        <string>Price:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QDoubleSpinBox" name="Price">
         <property name="minimumSize">
          <size>
           <width>95</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="maximum">
          <double>9999999.990000000223517</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="EuroSign">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>â‚¬</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="FromDateLabel">
       <property name="text">
        <string>Date From:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QDateEdit" name="FromDate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="displayFormat">
          <string>dd.MM.yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ToDateLabel">
       <property name="text">
        <string>Date To:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QDateEdit" name="ToDate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="displayFormat">
          <string>dd.MM.yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="FlightTab">
      <attribute name="title">
       <string>Flight</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="6" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="ToPlaceLabel">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>To:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="FromPlaceLabel">
         <property name="text">
          <string>From:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Airline:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="fromAirportLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="AirlineClass">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="From">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="ClassLabel">
         <property name="text">
          <string>Class:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="Airline">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="To">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="toAirportLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Airport:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Airport:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RentalcarTap">
      <attribute name="title">
       <string>Rental Car</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QLabel" name="FromPlaceLabel_3">
         <property name="text">
          <string>Pickup Location:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Vehicle Class:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="FromCar"/>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="TypeLabel_3">
         <property name="text">
          <string>Company:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="ToCar"/>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="ToPlaceLabel_3">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Return Location:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="Company">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="VehicleClass"/>
       </item>
       <item row="4" column="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="HotelTab">
      <attribute name="title">
       <string>Hotel</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Hotel:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Room Type:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="Hotel"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Town:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="Town"/>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="RoomType"/>
       </item>
       <item row="3" column="1" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>218</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TrainTab">
      <attribute name="title">
       <string>Train</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>From:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Layovers:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="FromTrain"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Departure:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QTimeEdit" name="ToTime"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QTimeEdit" name="FromTime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Arrival:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>To:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="ToTrain"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="TicketType"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Ticket Type:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QListWidget" name="listWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DetailsWindowFlightRental</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DetailsWindowFlightRental</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Airline</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Price</sender>
   <signal>valueChanged(double)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FromDate</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ToDate</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Airline</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>From</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>To</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AirlineClass</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FromCar</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ToCar</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Company</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VehicleClass</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hotel</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Town</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RoomType</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FromTrain</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FromTime</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ToTrain</sender>
   <signal>textEdited(QString)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ToTime</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FromDate</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>585</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ToDate</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>607</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
